samples-lib/src/main/java/com/esri/arcgismaps/sample/sampleslib/components/MessageDialog.kt

```diff
@@ -0,0 +1,66 @@
+package com.esri.arcgismaps.sample.sampleslib.components
+
+import android.content.res.Configuration
+import android.util.Log
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
+
+/**
+ * Composable component to display a dialog with a message
+ */
+@Composable
+fun MessageDialog(
+    title: String,
+    description: String = "",
+    showDialog: Boolean,
+    onDismissRequest: () -> Unit
+) {
+    if (showDialog) {
+        Log.e("SampleAlertMessage", "$title: $description")
+        if (description.isNotEmpty()) {
```

Comment:
The reason for checking `if (description.isNotEmpty())` is because of a weird compose behavior I noticed with AlertDialogs: 

<table>
<tr>
<td> Description </td> <td> Screenshot </td>
</tr>
<tr>
<td> 
MessageDialog with description:

```kotlin
title = { Text(title) },
text = { Text(description) },
```
</td>
<td>

![Screenshot 2023-06-15 at 10 24 38 AM](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/25b5ff26-1bd5-4998-9c02-342eb9bd7722)

</td>
</tr>
<tr>
<td> 
MessageDialog with no/empty description:

```kotlin
// notice the blank gap between the title and dismiss button
title = { Text(title) },
text = { if (description.isNotEmpty()) Text(description) },
```

</td>
<td>

![Screenshot 2023-06-15 at 10 25 22 AM](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/14830a26-5c83-409c-b96e-43e373aec89f)

</td>
</tr>

<tr>
<td> 
MessageDialog with no text attribute provided for AlertDialog:

```kotlin
// no gap between title and dismiss button
title = { Text(title) },
```

</td>
<td>

![Screenshot 2023-06-15 at 10 25 42 AM](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/53de1628-87a7-40ab-8934-04a3df53b14e)

</td>
</tr>
</table>

So as a weird workaround, to be table to display a dialog without the blank gap of an empty description, I've split the compose function into two `AlertDialog`'s, one for a dialog with a description and the other for a dialog without a description. I've removed the `if` check on line 32 as the description will always be not empty:
```kotlin
text = { if (description.isNotEmpty()) Text(description) }, // before 
text = { Text(description) }, // changed
```

Reply:
I see, that makes sense, very confusing on Android's part!